src/common.ts

```diff
@@ -110,12 +112,13 @@ export class LoggingCommon {
   readonly logName: string;
   private inspectMetadata: boolean;
   private levels: {[name: string]: number};
-  stackdriverLog: Log;
+  stackdriverLog: LogSeverityFunctions;
```

Comment:
Let's rename `stackdriverLog` to be `log` to avoid mentioning "stackdriver"

Reply:
I definitely can do it, but bear in mind that this variable already exposed (e.g. not private) and it might be accessed by user's code. Given a fact that it would be not a correct behavior, I will make this change.